hazelcast/src/test/java/com/hazelcast/quorum/QuorumTest.java

```diff
@@ -227,6 +229,49 @@ public boolean apply(Collection<Member> members) {
         }
     }
 
+    @Test
+    public void givenQuorumFunctionConfigured_whenImplementsHazelcastInstanceAware_thenHazelcastInjectsItsInstance() throws Exception {
```

Comment:
I can't see it on GitHub, but if the tests don't need it you could remove the `throws Exception` to shorten the lines a bit (the test names are already quite long).